index.js

```diff
@@ -41,6 +42,19 @@ function ldnode (argv) {
     // Setting options as local variable
     app.locals.ldp = opts;
 
+    // Setting CORS
+    app.use(cors({
+        methods: [
+            'OPTIONS', 'HEAD', 'GET',
+            'PATCH', 'POST', 'PUT', 'MKCOL', 'DELETE',
+            'COPY', 'MOVE', 'LOCK', 'UNLOCK'
```

Comment:
Are all those methods supported by ldnode? MKCOL, COPY, MOVE, LOCK, UNLOCK are methods specific to webdav.